plugins/inputs/rabbitmq/rabbitmq.go

```diff
@@ -275,11 +282,20 @@ func gatherOverview(r *RabbitMQ, acc telegraf.Accumulator) {
 		return
 	}
 
-	if overview.QueueTotals == nil || overview.ObjectTotals == nil || overview.MessageStats == nil {
+	if overview.QueueTotals == nil || overview.ObjectTotals == nil || overview.MessageStats == nil || overview.Listeners == nil {
 		acc.AddError(fmt.Errorf("Wrong answer from rabbitmq. Probably auth issue"))
 		return
 	}
 
+	var cluster_nodes, running_nodes int64 = 0, 0
+	for _, listener := range overview.Listeners {
+		if listener.Protocol == "clustering" {
+			cluster_nodes++
+		} else if listener.Protocol == "amqp" {
+			running_nodes++
```

Comment:
This does not appear to be an accurate way of measuring running nodes. I shut down one of the nodes in my cluster, and I still have 3 nodes with `"protocol": "amqp"`

Reply:
Ack, nevermind. error on my side. this does work